lib/templates.js

```diff
@@ -84,29 +115,40 @@ module.exports = function(client) {
       };
       return client.delete(options).asCallback(callback);
     },
-    preview: function(options, callback) {
+    /**
+     * Preview the most recent version of an existing template by id.
+     *
+     * @param {string} id
+     * @param {Object} options
+     * @param {RequestCb} [callback]
+     * @returns {Promise}
+     */
+    preview: function(id, options, callback) {
       var reqOpts;
       options = options || {};
 
-      if(!options.id) {
+      // Handle optional options argument
+      if (typeof options === 'function') {
```

Comment:
lol. it's fine if we do it across the board. i'm looking at subset which is why it's confusing to me.